src/App.tsx

```diff
@@ -8,12 +8,8 @@ function App() {
     return (
         <>
             <HeaderComponent />
```

Comment:
Please rename the component files from `About.component.tsx`, `Footer.component.tsx`, and `Header.component.tsx` to `About.tsx`, `Footer.tsx`, and `Header.tsx`. 
Additionally, adjust the component names within these files accordingly—change 
`const AboutComponent = ...` to `const About = ...`, 
and similarly for `Footer` and `Header`. 
Including 'component' in the filenames and variable names is redundant since the file extension `.tsx` and the React component pattern already imply that these are components. This change will make the code cleaner and more in line with typical JavaScript naming conventions.


https://github.com/MastersAcademy/fe-react-2024/wiki/%5BNaming%5D-How-to-make-a-name-for-your-variable,-method,-model